msi/docs/build-msi-locally.md

```diff
@@ -0,0 +1,139 @@
+Should you ever need to rebuild a Jenkins MSI on your Windows machine, here is a way to do it.
+
+# Pre-requisites
+
+## Jenkins WAR file
+
+First of all, you should get the Jenkins war file that will be inside that MSI file.
+You can get it from the official Jenkins website or from the Jenkins update center.
+I will use the official Jenkins website in this article.
+
+Check [Jenkins download page](https://www.jenkins.io/download/) and download the latest weekly version of Jenkins for example.
+For me, it is [jenkins.war](https://get.jenkins.io/war/2.380/jenkins.war) version 2.380.
+
+Change the URL according to the version you want to download.
+
+## Git tool
+
+You have a few options here:
+* Install [Git for Windows](https://community.chocolatey.org/packages/git)
+* Use your IDE to clone the repository
+* Use [GitHub Desktop](https://desktop.github.com/)
+* Use [SourceTree](https://www.sourcetreeapp.com/)
+* Use Cygwin integrated [Git command](https://cygwin.com/packages/summary/git.html)
+* [...]
+
+## Install MSBuild
+
+You can install [MSBuild](https://aka.ms/vs/17/release/vs_BuildTools.exe) from Visual Studio or from the [Build Tools for Visual Studio](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
+
+This command line tool is used to build the MSI file.
+
+## Install .NET Framework 3.5
+
+You may already have it installed on your machine, but not activated.
+You can activate it from the Windows Features dialog box.
+
+To access this dialog box, smash the `WinKey` plus `R`, then enter the command `appwiz.cpl` and push enter.
```

Comment:
```suggestion
To access this dialog box, smash `<kbd>⊞ Win</kbd> + <kbd>R</kbd>, then enter the command `appwiz.cpl` and push enter.
```
GitHub highlights the keyboard element with a specific button-like design.

Reply:
Not sure if there is a better display on your side but here is how this square character is rendered on Mac (seems unrecognized?):
<img width="191" alt="image" src="https://user-images.githubusercontent.com/91831478/220894291-ce1fe445-b238-4737-90cd-9eb6fc3caec3.png">

Also, in your suggestion the first backtick should be removed ;)

Reply:
Looks correct. It's unicode, similar to [⌘](https://www.compart.com/en/unicode/U+2318).